ddtrace/__init__.py

```diff
@@ -1,10 +1,18 @@
+import pkg_resources
+
 from .monkey import patch, patch_all
 from .pin import Pin
 from .span import Span
 from .tracer import Tracer
 from .settings import config
 
-__version__ = '0.27.0'
+
+try:
+    __version__ = pkg_resources.get_distribution(__name__).version
+except pkg_resources.DistributionNotFound:
+    # package is not installed
+    pass
```

Comment:
Should we offer some kind of fallback here?

e.g. `__version__ = 'unknown'` or `__version__ = '0.0.0'` or `__version__ = None` ?

Reply:
That sounds fair enough. Let's go for `None`!